demo/nested.html

```diff
@@ -16,9 +16,23 @@ <h1>Nested grids demo</h1>
       Use v9.2 <b>sizeToContent:true</b> on first subgrid item parent to grow/shrink as needed, while leaving leaf green items unchanged.<br>
       Uses v3.1 API to load the entire nested grid from JSON.<br>
       Nested grids uses v5 <b>column:'auto'</b> to keep items same size during resize.</p>
-    <a class="btn btn-primary" onClick="addNested()" href="#">Add Widget</a>
+    <div style="display: flex; flex-direction: row; gap: 5px;">
+    <a class="btn btn-primary" onClick="addWidget()" href="#">Add Widget</a>
     <a class="btn btn-primary" onClick="addNewWidget('.sub1')" href="#">Add Widget Grid1</a>
     <a class="btn btn-primary" onClick="addNewWidget('.sub2')" href="#">Add Widget Grid2</a>
+    <a class="btn btn-primary" onClick="addNested()" href="#">Add Widget With Nested Content</a>
+    <div style="width: 100px;">
+      <div
+          class="gridstack-add-widget grid-stack-item ui-draggable"
```

Comment:
this isn't the right way (and event handler below) to drag a nested grid... but I will commit your change and modify afterward.

Reply:
I've commited my changes this morning to the demo... 'dropped' event is better suited, but you would need the custom attr if you have more than 1 item type to drag in (not case of demo).